suave/sol/standard_peekers/example.sol

```diff
@@ -44,6 +44,44 @@ contract ExampleEthCallSource {
 
         return abi.encodeWithSelector(this.emptyCallback.selector);
     }
+
+    event LogCallback(bytes data);
```

Comment:
I'd suggest maybe a different name, since this doesn't actually cause anything to be executed, but is simply emitted
Maybe LogTrace? OffchainLogs?